### ISSUE TITLE
Unite to short lines in function calls?

### ISSUE THREAD

**Issue by artsiomkaltovich** [astral-sh/ruff#15055]
Here’s a polished version of your text:  

---

Apologies if this has already been discussed, but I couldn’t find any issue addressing this case.  

Would it make sense to unify code like this:  

```python
f(1, 
  2, 
  3)
```  

and produce the following result instead?  

```python
f(1, 2, 3)
```  

Currently, this code doesn’t produce any warnings: [Example](https://play.ruff.rs/be74ce4c-5643-43b8-a75b-4abcc1d4a498)

**Comment by MichaReiser** [comment]
You can use `ruff format` to format your code. You can see the formatted code in the [right panel](https://play.ruff.rs/be74ce4c-5643-43b8-a75b-4abcc1d4a498?secondary=Format)

**Comment by artsiomkaltovich** [comment]
Hello @MichaReiser Where can I find the command executed for format? Unfortunately, it doesn't work on my project.

**Comment by MichaReiser** [comment]
See https://docs.astral.sh/ruff/formatter/

**Comment by artsiomkaltovich** [comment]
Thank you for your help
